Sources/ContentManager.swift

```diff
@@ -68,6 +70,8 @@ public enum DTGError: LocalizedError {
             return "The item (id: \(itemId)) of the action was not found"
         case .invalidState(let itemId):
             return "try to make an action with an invalid state (item id: \(itemId))"
+        case .metadataLoading(itemId: _):
+            return "Item is already in the process of loading metadata"
```

Comment:
If the item id is already sent, use it, and return it in the string, same as in the other states.
More data is better.